.github/workflows/ci.yml

```
@@ -0,0 +1,92 @@
+name: CI Pipeline
+
+permissions:
+  checks: write
+  contents: read
+
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)